features/admin.users.v1/components/user-profile.tsx

```diff
@@ -1327,6 +1499,743 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
+                                .push(t("users:forms.validation.formatError", {
+                                    field: fieldName
+                                }));
+                        } else {
+                            setIsMultiValuedItemInvalid({
+                                ...isMultiValuedItemInvalid,
+                                [schema.name]: false
+                            });
+                        }
+                    } }
+                    displayErrorOn = "blur"
+                    listen={ (values: ProfileInfoInterface) => {
+                        setTempMultiValuedItemValue({
+                            ...tempMultiValuedItemValue,
+                            [schema.name]: values.get(schema.name)
+                        });
+                    } }
+                    maxLength={
+                        fieldName.toLowerCase().includes("uri") || fieldName.toLowerCase().includes("url")
+                            ? ProfileConstants.URI_CLAIM_VALUE_MAX_LENGTH
+                            : (
+                                schema.maxLength
+                                    ? schema.maxLength
+                                    : ProfileConstants.CLAIM_VALUE_MAX_LENGTH
+                            )
+                    }
+                />
+                <div hidden={ !showAccordion }>
+                    <Accordion
+                        elevation={ 0 }
+                        className="multi-valued-accordion"
+                        data-componentid={ `${ testId }-profile-form-${ schema.name }-accordion` }
+                        expanded={ expandMultiAttributeAccordion[schema.name] }
+                        onChange={ () => setExpandMultiAttributeAccordion(
+                            {
+                                ...expandMultiAttributeAccordion,
+                                [schema.name]: !expandMultiAttributeAccordion[schema.name]
+                            }
+                        ) }
+                    >
+                        <AccordionSummary
+                            aria-controls="panel1a-content"
+                            expandIcon={ <ChevronDownIcon /> }
+                            id="multi-attribute-header"
+                            className="accordion-summary"
+                            data-componentid={ `${ testId }-profile-form-${ schema.name }-accordion-summary` }
+                        >
+                            <label
+                                className={ `accordion-label ${schema.name === MOBILE_NUMBERS_ATTRIBUTE
+                                    ? "mobile-label"
+                                    : null}`
+                                }
+                            >
+                                { accordionLabelValue }
+
+                            </label>
+                            {
+                                showVerifiedPopup(accordionLabelValue)
```

Comment:
Don't we need to use react states to dynamically display these elements? I'm only seeing JS variables used within this function

Reply:
These values are derived from profileInfo which is already managed by state. When verification happens, profileInfo gets updated, causing a re-render, and these values are recomputed automatically.